src/main/resources/fester_messages.xml

```diff
@@ -131,4 +131,8 @@
   <entry key="MFS-135">Regular expression '{}' does not match '{}'</entry>
   <entry key="MFS-136">Cannot retrieve object '{}' from S3: unknown type '{}'</entry>
   <entry key="MFS-137">Test GetCollectionHandler with a test manifest: {}</entry>
+  <entry key="MFS-138">Supplied S3 key '{}' does not match what would have been derived from the IIIF resource URI: '{}'</entry>
+  <entry key="MFS-139">'{}' unable to handle message '{}': unknown action '{}'</entry>
+  <entry key="MFS-140">FAIL--manifest '{}' not found when it should have been put</entry>
```

Comment:
"FAIL" seems redundant here since the message is displayed with a test failure. Same for the message below.

Reply:
The reason for it is that I was trying to be consistent with the style used in e.g. MFS-036, but I am happy to remove.